int32 type
int32 ID
int32 sign
float32 value
int32 waitTime
uint8 alarm
